zoo.js

```diff
@@ -0,0 +1,177 @@
+/* Bash script used to turn table into javascript object declarations:
+sed -i 's/| /animals[] = {\n    name: \"/' Newfile.txt
+sed -i 's/ | /\",\n    commonName: \"/' Newfile.txt
+sed -i 's/ | /\",\n    species: \"/' Newfile.txt
+sed -i 's/ | /\",\n    location: \"/' Newfile.txt
+sed -i 's/ | /\",\n    age: /' Newfile.txt
+sed -i 's/ | /,\n    image: \"/' Newfile.txt
+sed -i 's/ |/\",\n};/' Newfile.txt
+
+Tweed and Trip's age field is missing a space before the |, so those must be
+fixed before running the script. The indices must be input manually after
+the script. */
+
+var animals = [];
+
+animals[0] = {
```

Comment:
I don't think using explicit array indexes is a good idea.

I recommend either putting the objects inside the array initializer, or use the [`.push()` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push) on arrays.

Array initializer:

```
var animals = [{
  name: "Mooney",
  commonName: "Blue monkey",
  ...
}, {
  name: "Sami",
  ...
}];
```

Using Array.prototype.push:

```
var animals = [];
animals.push({
  name: "Mooney",
  commonName: "Blue monkey",
  ...
});
```